org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: /app/webappreader.sh
```

Comment:
Should not be a path

Reply:
just to clarify: this it the **name** of the executable, that will be located from the `PATH`. Which is by default, `/app/bin`.

So:
1. it's wrong because you put a full path
2. it's not even pointing into a location in the `PATH`.